include/tesla.hpp

```diff
@@ -770,7 +770,10 @@ namespace tsl {
             }
 
             virtual Element* requestFocus(Element *oldFocus, FocusDirection direction) override {
-                return this->m_contentElement->requestFocus(oldFocus, direction);
+                if (this->m_contentElement != nullptr)
+                    return this->m_contentElement->requestFocus(oldFocus, direction);
+                else
+                    return this->m_contentElement;
```

Comment:
Instead of returning this->m_contentElement, just return nullptr since it always will have that value then. Makes it more readable

Reply:
Updated PR with this change